server/tilesource/base.py

```diff
@@ -1349,6 +1370,16 @@ def tileIterator(self, format=(TILE_FORMAT_PIL, ), resample=False,
                 only the stride of the tiles.  This is a dictionary containing:
             x: the overlap on the left and right in pixels.
             y: the overlap on the top and bottom in pixels.
+            edges: if True, then the edge tiles will exclude the overlap
+                distance.  If unset or False, the edge tiles are full size.
+                    As an example, suppose an image that is 8 pixels across
+                (01234567) and a tile size of 5 is requested with an overlap of
+                2.  If the edges option is False (the default), the following
+                tiles are returned: 01234, 12345, 23456, 34567.  Each tile
```

Comment:
When we think of overlap 2, i would expect neighboring tiles to overlap by or have that amount of pixels in common. In this case, i would expect -- (01, 01234, 34567, 67) or (01234, 34567)